app/services/garden_service.rb

```diff
@@ -3,6 +3,11 @@ def self.garden_details(params)
     get_parsed_json("api/v1/gardens/#{params[:id]}")
   end
 
+  def self.new_garden(params, current_user_id)
+    response = conn.post("api/v1/gardens?user_id=#{current_user_id}&longitude=#{params[:longitude]}&latitude=#{params[:latitude]}&name=#{params[:name]}&private=#{params[:private]}&description=#{params[:description]}")
+    JSON.parse(response.body, symbolize_names: true)
+  end
```

Comment:
```suggestion
  def self.new_garden(params, current_user_id)
    response = conn.post("api/v1/gardens") do |req|
      req.params = params
      req.params[:user_id] = current_user_id
    end
    JSON.parse(response.body, symbolize_names: true)
  end
```

The above worked for me! I noticed the user_id was not included in the params and passed it separately. Had you tried that?